01/Table/TableBody.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import TableRow from "./TableRow";
+
+export default class TableBody extends React.Component {
+  render() {
+    const { tableItems } = this.props;
+
+    const rows = this.props.cells.map((r) => {
+      return <TableRow key={r.id} price={r.price} quantity={r.quantity} />;
+    });
+    return (
+      <tbody>
+        <tr>{rows}</tr>
```

Comment:
Coś inaczej wygląda ta tabela :P

![image](https://github.com/devmentor-pl/practice-react-composition/assets/55095257/4505337a-4e63-47b0-8200-f2302dc333ea)

Tutaj problem jest z tym, że masz tylko jeden `<tr>`, a to on wyznacza wiersz.
Dlatego wew. `TableRow` powinnaś zdefiniować `<tr>`, a nie w tym miejscu.